sdks/python/apache_beam/examples/inference/multi_language_inference/README.md

```diff
@@ -0,0 +1,58 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+## Setting up the Expansion service
+Because we can not add local packages in Beam 2.43 we must create our own expansion service.
```

Comment:
@chamikaramj what wording makes the most sense here? Is this coming in 2.44/45, or should we just drop `in Beam 2.43`? If not, can we add what release this is coming in?